app/views/items/show.html.erb

```diff
@@ -0,0 +1,18 @@
+
+<% if flash[:notice] %>
+  <div class="flash notice"><%= flash[:notice] %></div>
```

Comment:
what is this if statement doing?

Reply:
same question like Odell

Reply:
I think if is optional. but I think it would be easy to read with if.